src/components/filmDescription/FilmDescription.js

```diff
@@ -23,16 +29,35 @@ const FilmDescription = ({film}) => {
           {overview}
         </p>
       </div>
+      <Button
+        id='closeDescription'
+        title='Close description'
+        disabled={false}
+        btnClass='btn--primary'
+        callback={() => {closeDescription()}}
```

Comment:
it is a good practice to move arrow function from JSX. It cam influence performance, because new function creates each time, when components re-renders.

Reply:
done